### DISCUSSION TITLE
Score & Streaks on Frontpage

### DISCUSSION THREAD

**Discussion by HariprasadManimozhi** [iSoron/uhabits#1097] [Feature Requests]
Hi, 
The current front page could be improvised by adding the metrics - Score % and Streaks. This enables to have a quick overview of habit loops on the front page at a glance and keep up the habit loops.

Attached a sample screenshot of the feature. Kindly include this feature in the future roadmap.

<img src="https://user-images.githubusercontent.com/28645647/132343133-a04d9322-6249-4243-aaca-49d2be73e0dd.png" width="350" height="600">

**Comment by hiqua** [comment]
I think there should be a discussion about which screens are big enough for this feature to make sense.

The current UI is also nice because it's rather clean, I'm not sure we want to put more stuff there.

**Reply by kflpk** [reply]
But couldn't it be an optional feature, so that people who want to have cleaner UI/have smaller screens could keep it disabled?

**Comment by ganesshkumar** [comment]
One way I am thinking is that it could look like and feel like one another habit (automatically created) which shows the percent of habits completed for that particular day.

![image](https://user-images.githubusercontent.com/2135089/132601462-a440df47-1011-41b8-bf1f-d10c9abd5c90.png)
